modules/gdscript/tests/gdscript_test_runner.cpp

```diff
@@ -340,27 +349,21 @@ void GDScriptTestRunner::handle_cmdline() {
 	// TODO: this could likely be ported to use test commands:
 	// https://github.com/godotengine/godot/pull/41355
 	// Currently requires to startup the whole engine, which is slow.
```

Comment:
Could remove this comment as well, it's not relevant anymore (unless we want to move the generation command to the test system too, but I don't think it's needed).

Reply:
Done :)